constants/objectives/results.py

```diff
@@ -51,6 +51,7 @@
         ResultType(31, "Learn Spells", "Learn {} Spells", list(range(1, len(id_spell) + 1))),
         ResultType(32, "Forget Spells", "Forget {} Spells", list(range(1, len(id_spell) + 1))),
         ResultType(33, "Max Morph Duration", "Max Morph Duration", None),
+        ResultType(59, "Magitek Upgrade", "Magitek Upgrade", None),
```

Comment:
Maybe we could add this to line 87 so we do not have this issue with a "running id"
```
category_types = {
   // keep as is
}

category_types["Command"] += ResultType(59, "Magitek Upgrade", "Magitek Upgrade", None)

Reply:
thanks, modified. Needed to use .append rather than +=, otherwise I got this error: 

```
  File "C:\Users\mtkm\WorldsCollide\constants\objectives\results.py", line 98, in <module>
    id_type[_type.id] = _type
AttributeError: 'int' object has no attribute 'id'
```